packages/vote/statement/registry.go

```diff
@@ -181,6 +183,14 @@ func (v *View) AddTimestamps(timestamps Timestamps) {
 	}
 }
 
+// UpdateLastStatementReceivedTime updates last statement issuing time in node's View.
+func (v *View) UpdateLastStatementReceivedTime(statementIssuingTime time.Time) {
+	v.lstMutex.RLock()
+	defer v.lstMutex.RUnlock()
+
+	v.LastStatementReceivedTimestamp = statementIssuingTime
```

Comment:
Maybe it's better to stick with statementReceivingTime instead of statementIssuingTime as we are relaying only on the time when statement was read by our node, so this could be misleading?

Reply:
Good point ;)